src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -38,10 +38,7 @@ import Halogen.Subscription as HS
 import Unsafe.Reference (unsafeRefEq)
 
 type Renderer r
-  = forall s f act ps i o
-   . Ref LifecycleHandlers
-  -> Ref (DriverState r s f act ps i o)
-  -> Effect Unit
+  = forall s f act ps i o. Ref LifecycleHandlers -> Ref (DriverState r s f act ps i o) -> Effect Unit
```

Comment:
This looks like a bug?

Reply:
I was about to comment on that. This doesn't look like it's the correct behavior.